### ISSUE TITLE
Arrows bug

### ISSUE THREAD

**Issue by StephanKup** [studio-minus/ppg-bugs-and-requests#1356]
**Describe the bug**
Okay so, I just noticed that arrows fly through the human, they don't stuck inside of him and only after some time an arrow finally hits the human

**Clearly describe how to reproduce the bug**
Just shoot with the arrow gun

**Mod list**
None

**System info (please complete the following information)**
 - Operating system: Windows10
 - Game version: Latest
 - GPU: NVIDIA GeForce GT 240
 - CPU: Pentium(R) Dual-Core  CPU      E5500  @ 2.80GHz   2.80 GHz

**Comment by DindinYT37** [comment]
I noticed this for some time now, I thought a mod had caused it though - apparently not.

**Comment by SeVeredC0nnection** [comment]
If your collision quality is set to discrete try setting it to continuous. That could be causing the issue.

**Comment by DindinYT37** [comment]
> If your collision quality is set to discrete try setting it to continuous. That could be causing the issue.

You misunderstood the issue.

**Comment by StephanKup** [comment]
> If your collision quality is set to discrete try setting it to continuous. That could be causing the issue.

No, it's set to dynamic so it's not the issue, something's wrong with the arrows, maybe because of their speed they go through the human, but I'm not a dev so I don't really know

**Comment by Drogo-man** [comment]
They are set to pierce no matter what (to what I assume is to make them not clip). This might also be why they just glide out of them.

**Comment by Gigi-7688** [comment]
If the quality is set to dynamic, small objects moving at high speed can pass through things.

**Comment by xpdatabase11** [comment]
I also noticed when holding the arrows off the ground, the rotation starts to add up and just starts to spin on their own. Like there is negative mass.
